### PR TITLE
chore: add mainnet assets for running nodes

### PR DESCRIPTION
**Pull request by owen-reorg**
The Mainnet assets for running full nodes.

Check the [doc](https://docs.bnbchain.org/opbnb-docs/docs/tutorials/running-a-mainnet-node) for the detailed guide.

### CODE REVIEW

File: assets/mainnet/rollup.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  "block_time": 1,
+  "max_sequencer_drift": 600,
+  "seq_window_size": 14400,
+  "channel_timeout": 1200,
+  "l1_chain_id": 56,
+  "l2_chain_id": 204,
+  "regolith_time": 0,
+  "batch_inbox_address": "0xff00000000000000000000000000000000000204",
+  "deposit_contract_address": "0x1876ea7702c0ad0c6a2ae6036de7733edfbca519",
+  "l1_system_config_address": "0x7ac836148c14c74086d57f7828f2d065672db3b8"
```
Comment (selarun13) [inline assets/mainnet/rollup.json:28]: 0x6F99E90Dd385c23826b416528e325214671FeBf6

File: assets/mainnet/rollup.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+}
```
Comment (selarun13) [inline assets/mainnet/rollup.json:29]: 0x6F99E90Dd385c23826b416528e325214671FeBf6

**Review by selarun13 — APPROVED** [review]

File: assets/mainnet/rollup.json (hunk 1 above)
Comment (owen-reorg) [reply assets/mainnet/rollup.json:28]: > 0x6F99E90Dd385c23826b416528e325214671FeBf6

This is the implementation address. The proxy address is `0x7ac836148c14c74086d57f7828f2d065672db3b8`.

**Review by Lredhdx — APPROVED** [review]

File: assets/mainnet/rollup.json (hunk 2 above)
Comment (selarun13) [inline assets/mainnet/rollup.json:29]: 0x6F99E90Dd385c23826b416528e325214671FeBf6

**Review by selarun13 — COMMENTED** [review]
0x6F99E90Dd385c23826b416528e325214671FeBf6

my adress

File: assets/mainnet/rollup.json (hunk 2 above)
Comment (selarun13) [inline assets/mainnet/rollup.json:29]: 0x6F99E90Dd385c23826b416528e325214671FeBf6

**Review by selarun13 — COMMENTED** [review]
0x6F99E90Dd385c23826b416528e325214671FeBf6

**Pull request merged**